db_schema/schema.sql

```diff
@@ -242,3 +242,253 @@ CREATE TABLE IF NOT EXISTS updates (
 CREATE EVENT IF NOT EXISTS updates_cleanup
 ON SCHEDULE EVERY 1 DAY
 DO DELETE FROM updates WHERE created < DATE_SUB(NOW(), INTERVAL 7 DAY);
+
+CREATE TABLE IF NOT EXISTS countries (
+  id INTEGER PRIMARY KEY,
+  name VARCHAR(255) NOT NULL,
+  code CHAR(2) NOT NULL,
```

Comment:
do we know more about the code and what ccc means? is it any standard? what about the translation of the table? do we have german names as well?

Reply:
seems to be Apple-internal terminology. We could remove it.
No German translations yet. I could add that in a future version.